code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -826,3 +826,29 @@
 /datum/reagent/bottle_lightning/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	if(prob(25))
 		tesla_zap(M, 6, 1500)
+
+//Relevant for Cult
+/datum/reagent/blood_of_narsie
+	name = "Blood of Nar-Sie"
```

Comment:
I feel like renaming it "Corrupted Blood" would be more appropriate here. What if people don't want to follow the whole Nar'sie thread? Using more generic names might be a better idea.